src/internal/modules/reporting/controllers.js

```diff
@@ -0,0 +1,34 @@
+const services = require('../../lib/connectors/services');
+
+/**
+ * Loads the Charging Reports page
+ * @param request {Object} The request object
+ * @param h {Object} The response object
+ * @returns {Object} h.view Object containing page details
+ */
+const getChargingForecastReportsPage = (request, h) => {
+  return h.view('nunjucks/reporting/charging-forecast-reports', {
+    ...request.view,
+    back: '/manage',
+    pageTitle: 'Download a charging forecast report'
+  });
+};
+
+/**
+ * Loads a specific report by making a request to the backend
+ * @param request {Object} The request object
+ * @param h {Object} The response object
+ * @returns {Object} h.response Response body
+ */
+const getDownloadableReport = async (request, h) => {
+  const { reportIdentifier } = request.params;
+  // get signed url
+  const report = await services.water.reporting.getReport(request, h);
```

Comment:
Although I don't think it's causing an issue, I don't think the `await` is needed here

Reply:
I THINK it's needed - I found from experimenting with it that if I omit it, only the first chunk is downloaded 😁